relayer/src/relayer.rs

```diff
@@ -224,6 +224,32 @@ pub enum RelayerError {
 
 pub type RelayerResult<T> = Result<T, RelayerError>;
 
+pub struct RelayerHandle {
+    packet_subscriptions:
+        Arc<RwLock<HashMap<Pubkey, TokioSender<Result<SubscribePacketsResponse, Status>>>>>,
```

Comment:
use Arc<Dashmap> instead of Arc<RwLock<Hashmap>>?

Reply:
this is an updating handle to subscriptions. grab a handle once and always be able to access the newest subscribers

Reply:
github ate the angle brackets, updated now

Reply:
i like the Dashmap here too.  it avoids all of the manual lock management below.